packages/editor-preview/README.md

```diff
@@ -0,0 +1,7 @@
+# Theia - Editor Preview Extension
+
```

Comment:
It would be nice to have a description of what this extension brings. Maybe some bits from the overview part from this comment could be copied:  https://github.com/theia-ide/theia/issues/1252#issuecomment-428766722

Reply:
https://github.com/theia-ide/theia/pull/3561